examples/Golang/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/Be-MobileNV/fcd-endpoint-client/examples/Golang
 
-go 1.18
+go 1.22
+
+replace github.com/Be-MobileNV/fcd-endpoint-client/client/Golang => github.com/TomAlard/fcd-endpoint-client/client/Golang v0.0.0-20250422093010-1ba02894ddbc
```

Comment:
The example Go code imports this repository from Github as a source. This is needed for it to access the new code. Obviously this should be removed, but I think we can only do that in a second PR?